### DISCUSSION TITLE
Can't create another record after creating one

### DISCUSSION THREAD

**Discussion by tan911** [prisma/prisma#23524] [Q&A] [answered]
### Question

Sorry, I'm new to prisma started to learn about this orm. 
My question is, What wrong with this model why this wont allow me to create another column in project table?
I mean if I try to create a project record base on the User id it throws an error that looks like this:

`Invalid prisma.project.create() invocation in`
`Unique constraint failed on the fields: (project_id) `

### Expected behavior (optional)

I should be able to create multiple record column to project table base on the User table id

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
model User {
  id            String    @id @default(cuid())
  name          String?   @db.VarChar(45)
  email         String    @unique @db.VarChar(50)
  emailVerified DateTime? @map("email_verified")
  password      String?
  createdAt     DateTime  @default(now()) @map("created_at")
  image         String?
  accounts      Account[]
  profile       Profile?
  about         About?
  work          Work[]
  project       Project[]
  contact       Contact?
}

model Project {
  id          Int      @id @default(autoincrement())
  user        User     @relation(fields: [projectId], references: [id])
  projectId   String   @unique @map("project_id")
  name        String   @db.VarChar(50)
  status      Status   @default(Active)
  description String
  image       String
  techStack   String[] @map("teck_stack")
  updatedAt   DateTime @default(now()) @map("updated_at")
}

```

Mutation:
```ts
createProjects: protectedProcedure.input(createProjectSchema).mutation(async (opts) => {
        try {
            const { input, ctx } = opts
            await prisma.project.create({
                data: {
                    user: { connect: { id: ctx.user.id } },
                    name: input.name,
                    description: input.description,
                    status: input.status,
                    techStack: [input.techStack],
                    image: input.image,
                },
            })

            return { message: 'Project created successfully!' }
        } catch (error) {
            throw new TRPCError({
                code: 'INTERNAL_SERVER_ERROR',
                message: 'Failed to create a project',
            })
        }
    }),

```

- OS: Windows 
- Database: PostgreSQL
- Node.js version: 18.17.0

- Prisma version: 5.9.1

**Comment by tan911** [comment]
I should consider researching more before positing here. So after a lot of research I think the problem here is a unique constraint.

**Reply by ludralph** [reply, marked as answer]
Hi @tan911 👋 

You are correct that the error message you're seeing is due to a unique constraint violation on the `projectId` field in your `Project` model. This means that you're trying to create a new Project with a `projectId` that already exists in the database.
This means that each Project must have a unique `projectId`. If you try to create a new Project with a `projectId` that's already been used, Prisma will throw an error.
